pkgs/servers/x11/xorg/default.nix

```diff
@@ -3135,34 +3135,14 @@ self: with self; {
     };
   })) {};
 
-  # THIS IS A GENERATED FILE.  DO NOT EDIT!
-  xf86videonouveau = callPackage ({ stdenv, pkg-config, fetchurl, xorgproto, libdrm, udev, libpciaccess, xorgserver, testers }: stdenv.mkDerivation (finalAttrs: {
```

Comment:
Ofborg is failing to eval because `xf86videonouveau` was removed

Reply:
Yup that's what I get for not reviewing the diffs after modifying the script.